src/BenchmarkDotNet/Diagnosers/UnresolvedDiagnoser.cs

```diff
@@ -38,6 +31,9 @@ public class UnresolvedDiagnoser : IDiagnoser
         public IEnumerable<ValidationError> Validate(ValidationParameters validationParameters)
             => new[] { new ValidationError(false, GetErrorMessage()) };
 
-        private string GetErrorMessage() => $"Unable to resolve {unresolved.Name} diagnoser. {ErrorMessage}";
+        private string GetErrorMessage() => $@"Unable to resolve {unresolved.Name} diagnoser. 
+            {(RuntimeInformation.IsFullFramework 
+                ? "Please make sure that you have installed the latest BenchmarkDotNet.Diagnostics.Windows package." 
+                : "To use the classic Windows diagnosers for .NET Core you need to run the benchmarks for desktop .NET. More info: http://adamsitnik.com/Hardware-Counters-Diagnoser/#how-to-get-it-running-for-net-coremono-on-windows")}";
```

Comment:
Can we mirror this information on Github wiki pages?

Reply:
I think it would be better to put this information in the documentation instead of GitHub wiki.

Reply:
another good catch! This info is obsolete now because we can load the Diagnostics.Windows package from .NET Core app on Windows now. Fixed in https://github.com/dotnet/BenchmarkDotNet/pull/717/commits/0cccba1c5761b07f06332a8a945997c92207106d